numpyro/mcmc.py

```diff
@@ -270,7 +260,8 @@ def _nuts_next(step_size, inverse_mass_matrix, vv_state, rng):
                                  max_tree_depth=max_treedepth)
         accept_prob = binary_tree.sum_accept_probs / binary_tree.num_proposals
         num_steps = binary_tree.num_proposals
-        vv_state = vv_state.update(z=binary_tree.z_proposal,
+        vv_state = IntegratorState(z=binary_tree.z_proposal,
+                                   r=vv_state.r,
```

Comment:
Is this added for logging purposes?

Reply:
This is added because namedtuple does not have `update` method. Here, `r` field keeps its old value.

Reply:
Ahh..sorry, I missed that. Makes sense.